src/app/content/components/Toolbar/StudyGuidesButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import styled, { css } from 'styled-components/macro';
+import studyGuidesIcon from '../../../../assets/studyGuidesIcon.svg';
+import theme from '../../../theme';
+import { enableStudyGuides } from '../../selectors';
+import { toolbarIconStyles } from './iconStyles';
+import { PlainButton, toolbarDefaultText } from './styled';
+
+// tslint:disable-next-line:variable-name
+export const StudyGuidesWrapper = styled(PlainButton)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 2rem;
+  height: auto;
+  ${theme.breakpoints.mobile(css`
+    margin-right: 0;
+  `)}
+`;
+
+// tslint:disable-next-line:variable-name
+const StudyGuidesIcon = styled.img`
+  ${toolbarIconStyles}
+  padding: 0.2rem;
```

Comment:
what is this padding for?

Reply:
`toolbarIconStyles` has padding set to `0.4rem` but in this case, an icon was too small comparing tot he design